hw/ip/aes/pre_sca/alma_post_syn/simplify_techlib.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+# This script reads three files [1], [2], [3] and constructs file [4], such that:
+# [1] Original techlib verilog file
+# [2] Manually written patch file that contains new cell definitions
+# [3] The AES S-box netlist file (consisting of TSMC cells)
```

Comment:
AES S-box netlist file -> synthesized netlist file ? because the whole flow is applicable to any circuit :-)

Reply:
Aes S-box netlist file -> synthesized netlist file is OK.

However, I am not sure about dropping TSMC references, because this scrips uses a a regexp for cell names of this library. It would not work for the others right out of the box.

Reply:
We've discussed this offline and agreed that it's better to remove these references.